content/en/docs/started/k8s/overview.md

```diff
@@ -33,6 +30,9 @@ The Kubernetes cluster must meet the following minimum requirements:
   - Older versions of Kubernetes may not be compatible with the latest Kubeflow versions. The following matrix
     provides information about compatibility between Kubeflow and Kubernetes versions.
 
+* A default Storage Class with a Dynamic Volume Provisioner.
+  - Multiple components like Kubeflow Pipelines and Jupyter Web App assume the existence of a volume provisioner.
```

Comment:
Suggestions:

- Would it make sense to reword this part while also informing users who aren't proficient at all things k8s what StorageClass and DVP are? (Also, I think `StorageClass` <- `Storage Class` AFAIK).

For instance:

```diff
The Kubernetes cluster must meet the following minimum requirements:

- - Your cluster must include at least one worker node with a minimum of:
+ * Your cluster must include at least one worker node with a minimum of:
...

* The recommended Kubernetes version is {{% kubernetes-tested-version %}}.
...

- * A default Storage Class with a Dynamic Volume Provisioner.
+ * You need a default
+   [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+   with a [dynamic volume
+   provisioner](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
+   For more information, refer to [this
+    guide](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/#before-you-start).
...
```

Perhaps, it may be even more helpful to add a numerical sequence, such as:

```
The Kubernetes cluster must meet the following minimum requirements:

1. Your cluster must include at least one worker node with a minimum of:
...

2. The recommended Kubernetes version is {{% kubernetes-tested-version %}}.
...

3. You need a default [StorageClass](...) with a [dynamic volume provisioner](...).
...
```

These are just suggestions.

WDYT?